build.ps1

```diff
@@ -48,6 +48,9 @@ function bazel() {
 # which later causes issues on Bazel init (source forest creation) on Windows. A shutdown closes workers,
 # which is a workaround for this problem.
 bazel shutdown
+# Temporary to workaround rules_nodejs update.
+bazel clean --expunge
```

Comment:
The removal of the unused (and deprecated) `@bazel/bazel` package requires this. I’ll merge this later today to minimize the time we have this in.

Reply:
Meaning this can be undone after the CI workers are back to a consistent state?

Reply:
Exactly, I’ll undo it tomorrow.